buildSrc/src/main/groovy/InjectedExecOps.groovy

```diff
@@ -0,0 +1,28 @@
+import  org.gradle.process.ExecOperations
+import javax.inject.Inject
+
+/**
+ * This class is for calling out to the command line.  You probably don't need this class and can
+ * use an Exec task instead.
```

Comment:
Is this comment accurate?

Reply:
It was accurate, but I figured out how to not use this class at all, so I deleted it.